test/suites/standard1/kustomize.robot

```diff
@@ -244,3 +271,25 @@ Restore Default Config
         Restart MicroShift
         Sleep    10 seconds    # Wait for systemd to catch up
     END
+
+Verify Manifest Deletion
+    [Documentation]    Verify that manifest deletion works as expected.
+    [Arguments]    ${ns}    ${manifest_path}
+    ConfigMap Path Should Match    ${ns}    ${manifest_path}
+    ${delete_base_dir}    ${delete_manifest_path}=    Get Manifest Deletion Paths    ${manifest_path}
+    Append To List    ${TO_DELETE}    ${delete_base_dir}
+    Command Should Work    mkdir -p ${delete_base_dir}
+    Command Should Work    mv ${manifest_path} ${delete_manifest_path}
+    Restart MicroShift
+    ConfigMap Should Be Missing    ${ns}
+    Command Should Work
+    ...    journalctl -u microshift --grep "Deleting kustomization at ${delete_manifest_path} was successful."
```

Comment:
Please, use journal.py API to implement this. We need to save the cursor before restarting MicroShift and then check the pattern. This should assure we do not find old messages.